### PR TITLE
Updating release management scripts

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @victorhurdugaci

File: _use-release-management.shade [hunk 1]
```diff
@@ -43,24 +44,58 @@
...
                 {
                     File.WriteAllText(nugetConfigPath, modified);
                     GitCommand(repo, "add NuGet.config");
-                    GitCommand(repo, "commit -m \"Updating to release NuGet.config.\"");
+                    filesChanged = true;
                 }
             }
 
+            var buildPs1Path = Path.Combine(repo, "build.ps1");
+            if (File.Exists(buildPs1Path))
```
Comment (victorhurdugaci) [inline _use-release-management.shade]: I'd move the logic in this if block in a function so you don't duplicate it

File: _use-release-management.shade [hunk 2]
```diff
@@ -172,18 +207,18 @@
...
         IEnumerable<string> GetAllRepos()
         {
             var nonDefaultRepos = new[]
             {
-              "DNX",
+              "CoreCLR",
               "Coherence",
               "Coherence-Signed",
-              "dnvm",
-              "Setup",
```
Comment (victorhurdugaci) [inline _use-release-management.shade]: What happened to Setup?

**Comment by victorhurdugaci** [comment]
:shipit: after addressing the comments

File: _use-release-management.shade (hunk 2 above)
Comment (pranavkm) [inline _use-release-management.shade]: They don't want us to branch it

**Pull request closed without merging**